server/cluster/cluster.go

```diff
@@ -301,6 +301,26 @@ func (c *Cluster) RegisterNode(ctx context.Context, registeredNode RegisteredNod
 
 	c.registeredNodesCache[registeredNode.Node.Name] = registeredNode
 
+	// When the number of nodes in the cluster reaches the threshold, modify the cluster status to stable.
+	if uint32(len(c.registeredNodesCache)) >= c.metaData.MinNodeCount && c.topologyManager.GetClusterState() == storage.ClusterStateEmpty {
```

Comment:
Considering this situation here, we hope that the cluster will be initialized when the number of cluster nodes reaches the threshold, so as to avoid the allocation of shards when there is only one node, resulting in uneven distribution of shards.
Furthermore, when the state of the cluster is already stable and the entire cluster is restarted, we do not want to assign all shards to a node when a node is started. I leave a todo here.